charts/k8s-watcher/README.md

```diff
@@ -212,11 +213,11 @@ KOMOKW_REDACT_LOGS="password=(.+?)\b (?U)\"sessionId\": (\".+\"{1})"
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
-
 ### Using a Proxy
 
-Komodor's support the standard proxy environment variables (`HTTP_PROXY, HTTPS_PROXY, NO_PROXY`) as well as these variables prefixed by `KOMOKW_` which will assign the proxy only to the HTTP clients communicating with Komodor. This is useful in case you want to leave the communication to the Kubernetes API in-cluster.  
+Komodor's support the standard proxy environment variables (`HTTP_PROXY, HTTPS_PROXY, NO_PROXY`) as well as these variables prefixed by `KOMOKW_` which will assign the proxy only to the HTTP clients communicating with Komodor. This is useful in case you want to leave the communication to the Kubernetes API in-cluster.
```

Comment:
@erez-rabih If you already change it..Please fix this typo also

Komodor supports the standard....

Reply:
75288de